### ISSUE TITLE
Regarding Results after Epoch 1 Training

### ISSUE THREAD

**Issue by aviralchharia** [XunshanMan/MVGFormer#4]
Hi, thanks for the great work! I tried training the model and got two tables as the result after the first epoch. 

1. The two tables show different values of $AP_{25}$, $AP_{50}$, $AP_{75}$, $AP_{100}$, etc (Same for Recall values). Also two different values of $MPJPE$ are printed. I assume the paper reports the first table. What does the second table here represent? 
2. Moreover, wandB only shows the Table 2 values. Am I missing something?

![image](https://github.com/user-attachments/assets/38bc42e0-3abb-45b5-be2b-de2c523df7cc)

**Comment by XunshanMan** [comment]
Hi, thanks for the question! The difference between the two tables is whether to use NMS before outputting the results. 
NMS is necessary (see Table 1) because we start with 1024 queries (pose assumptions), and even after filtering by classification scores, multiple queries remain linked to the same person. After NMS, the redundant queries can be effectively filtered. For Wandb, you can add the metric in Table 1 into it for plot supervision.

**Comment by SWWdz** [comment]
> Hi, thanks for the great work! I tried training the model and got two tables as the result after the first epoch.
> 
> 1. The two tables show different values of 
>      A
>      
>        P
>        
>          25
>        
>      
>    , 
>      A
>      
>        P
>        
>          50
>        
>      
>    , 
>      A
>      
>        P
>        
>          75
>        
>      
>    , 
>      A
>      
>        P
>        
>          100
>        
>      
>    , etc (Same for Recall values). Also two different values of 
>      M
>      P
>      J
>      P
>      E
>     are printed. I assume the paper reports the first table. What does the second table here represent?
> 2. Moreover, wandB only shows the Table 2 values. Am I missing something?
> 
> ![image](https://private-user-images.githubusercontent.com/62457915/354934175-38bc42e0-3abb-45b5-be2b-de2c523df7cc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQzMTY0MTEsIm5iZiI6MTcyNDMxNjExMSwicGF0aCI6Ii82MjQ1NzkxNS8zNTQ5MzQxNzUtMzhiYzQyZTAtM2FiYi00NWI1LWJlMmItZGUyYzUyM2RmN2NjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIyVDA4NDE1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmMDI4MWJhZjk3MjM3OWFmOGJjMjAzMGI3YzMyMWI2MWFiYjUxZmY3MzI5OWU4N2YzMTc5YzY0ZWMwNjAwNGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.J2kp7OnTbEeH692D2wlQR_ZE4Z48mmXeUsNboQld7EM)

Can I ask you how to train this model? What's your cuda version, gpu and your environment?

**Comment by XunshanMan** [comment]
> > Hi, thanks for the great work! I tried training the model and got two tables as the result after the first epoch.
> > 2. Moreover, wandB only shows the Table 2 values. Am I missing something?
> > ![image](https://private-user-images.githubusercontent.com/62457915/354934175-38bc42e0-3abb-45b5-be2b-de2c523df7cc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQzMTY0MTEsIm5iZiI6MTcyNDMxNjExMSwicGF0aCI6Ii82MjQ1NzkxNS8zNTQ5MzQxNzUtMzhiYzQyZTAtM2FiYi00NWI1LWJlMmItZGUyYzUyM2RmN2NjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIyVDA4NDE1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmMDI4MWJhZjk3MjM3OWFmOGJjMjAzMGI3YzMyMWI2MWFiYjUxZmY3MzI5OWU4N2YzMTc5YzY0ZWMwNjAwNGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.J2kp7OnTbEeH692D2wlQR_ZE4Z48mmXeUsNboQld7EM)
> 
> Can I ask you how to train this model? What's your cuda version, gpu and your environment?

Please refer to the README Training part for the details: https://github.com/XunshanMan/MVGFormer?tab=readme-ov-file#2-training.

Here is a runnable environment for reference:

- Python 3.10.0
- PyTorch 2.0.0
- Cuda 11.7

**Comment by qiqiqaq404** [comment]
~/MVGFormer-master/models/下的预训练模型可以分享给我吗？
